docs/first-steps/post-deployment.md

```diff
@@ -138,6 +136,32 @@ Similarly to the management user's MFA enabling step, you are switching from usi
 
 As a last step you need to make sure that `MFA_ENABLED` is set to `true` in the `build.env` file.
 
+## Re-Configure profiles  MFA
+
+In order to everything works correctly, you need to set your aws config profiles with the correct mfa device configuration.
+
+Run again  `leverage credentials configure`  with the flag `--fetch-mfa-device` and select the option of `skip credentials configuration`.
+
+``` bash
+leverage credentials configure --fetch-mfa-device --type MANAGEMENT
+leverage credentials configure --fetch-mfa-device --type SECURITY
+```
+<pre><code>
+<span class="fsg-timestamp">[10:10:11.033]</span> INFO     Loading configuration file.
+<span class="fsg-timestamp">[10:10:11.092]</span> INFO     Loading project environment configuration file.
+<span class="fsg-timestamp">[10:10:11.093]</span> INFO     Loading Terraform common configuration.
+<span class="fsg-prompt">></span> <b> Credentials already configured for ld-management: <span class="fsg-userinput">Skip credentials configuration. Continue on with assumable roles setup.</span></b>
+<span class="fsg-timestamp">[10:10:30.345]</span> INFO     Attempting to fetch organization accounts.
+<span class="fsg-timestamp">[10:10:33.928]</span> INFO     Configuring assumable roles.
+<span class="fsg-timestamp">[10:10:33.932]</span> INFO     <b>Fetching MFA device serial.</b>
+<span class="fsg-timestamp">[10:10:37.473]</span> INFO     Backing up account profiles file.
+<span class="fsg-timestamp">[10:10:38.913]</span> INFO             Configuring profile <b>me-management-oaar</b>
+<span class="fsg-timestamp">[10:10:53.088]</span> INFO             Configuring profile <b>me-security-oaar</b>
+<span class="fsg-timestamp">[10:11:08.229]</span> INFO             Configuring profile <b>me-shared-oaar</b>
+<span class="fsg-timestamp">[10:11:23.185]</span> INFO     <b>Account profiles configured in:</b> <span class="fsg-path">/home/user/.aws/me/config</span>
+</code></pre>
+
+
```

Comment:
@rodriguez-matias Matu, very small one here, not sure if 2 lines are needed here, may be only 1 should be enough.

Reply:
done!